test/celeritas/optical/OpticalCollector.test.cc

```diff
@@ -223,35 +223,35 @@ auto LArSpherePreGenTest::run(size_type num_tracks, size_type num_steps)
     using ItemsRef
         = Collection<OpticalDistributionData, Ownership::reference, M>;
 
-    auto get_result = [&](PreGenResult& result,
-                          ItemsRef const& buffer,
-                          size_type size) {
-        // Copy buffer to host
-        std::vector<OpticalDistributionData> data(size);
-        Copier<OpticalDistributionData, MemSpace::host> copy_data{
-            make_span(data)};
-        copy_data(M, buffer[BufferRange(BufferId(0), BufferId(size))]);
-
-        std::set<real_type> charge;
-        for (auto const& dist : data)
-        {
-            result.total_num_photons += dist.num_photons;
-            result.num_photons.push_back(dist.num_photons);
-            if (!dist)
-            {
-                continue;
-            }
-            charge.insert(dist.charge.value());
-
-            auto const& pre = dist.points[StepPoint::pre];
-            auto const& post = dist.points[StepPoint::post];
-            EXPECT_GT(pre.speed, zero_quantity());
-            EXPECT_NE(post.pos, pre.pos);
-            EXPECT_GT(dist.step_length, 0);
-            EXPECT_EQ(0, dist.material.get());
-        }
-        result.charge.insert(result.charge.end(), charge.begin(), charge.end());
-    };
+    auto get_result
+        = [&](PreGenResult& result, ItemsRef const& buffer, size_type size) {
+              // Copy buffer to host
+              std::vector<OpticalDistributionData> data(size);
+              Copier<OpticalDistributionData, MemSpace::host> copy_data{
+                  make_span(data)};
+              copy_data(M, buffer[BufferRange(BufferId(0), BufferId(size))]);
+
+              std::set<real_type> charge;
+              for (auto const& dist : data)
+              {
+                  result.total_num_photons += dist.num_photons;
+                  result.num_photons.push_back(dist.num_photons);
+                  if (!dist)
+                  {
+                      continue;
+                  }
+                  charge.insert(dist.charge.value());
+
+                  auto const& pre = dist.points[StepPoint::pre];
+                  auto const& post = dist.points[StepPoint::post];
+                  EXPECT_GT(pre.speed, zero_quantity());
+                  EXPECT_NE(post.pos, pre.pos);
+                  EXPECT_GT(dist.step_length, 0);
+                  EXPECT_EQ(0, dist.material.get());
+              }
+              result.charge.insert(
+                  result.charge.end(), charge.begin(), charge.end());
+          };
```

Comment:
What changed here?

Reply:
Oh, the change in the `OpticalCollector` constructor from `const&` to `shared_ptr` (line 163) led to a full clang-formatting of the file.

Reply:
OK, makes sense. It can be reverted then.